int64 motor_no
---
float64 val
float64 enc
